projects/1.16/assets/ars-nouveau-gears/ars_gears/patchouli_books/worn_notebook/zh_cn/entries/ars_gears/starting_ars_gears.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Ars: G.E.A.R入门",
+  "icon": "ars_gears:combat/wood_combat_gear",
+  "sortnum":0,
+  "category": "ars_gears",
+  "pages": [
+    {
+      "type": "text",
+      "text": "Ars: G.E.A.R是新生魔艺的附属，为其加入了2个物品：战斗型多效装备和效用型多效装备。$(br2)这些物品会替代部分原版物品和新生魔艺的法术书。"
+    },
+    {
+      "type": "text",
+      "text": "$(bold)装备模式：$()它们共有3种模式，可用“循环装备”（$(k:ars_gears.cycle_gear)）键在各种模式间切换。$(br2)他人的多效装备若被你捡起，则会自动转变为你的多效装备（也包括耐久度）。"
+    },
+    {
+      "type": "text",
+      "text": "$(bold)修复：$()需打开多效工具的物品栏（$(k:ars_gears.gear_inventory)）并在其中放置食物。$(br2)食物的品质会影响回复的耐久度。$(br2)（$(bold)注意：$()饱和度和食物中毒不造成影响）"

```

```suggestion
      "text": "$(bold)修复：$()需打开多效工具的物品栏（$(k:ars_gears.gear_inventory)）并在其中放置食物。$(br2)食物的品质会影响回复的耐久度。$(br2)（$(bold)注意：$()饱和和食物负面效果不会影响）"

```
可能指的是饱和这个药水效果，和饥饿、中毒。如果你有时间进游戏看一下……我没配环境。

---

```suggestion
      "text": "$(bold)修复：$()需打开多效工具的物品栏（$(k:ars_gears.gear_inventory)）并在其中放置食物。$(br2)食物的品质会影响回复的耐久度。$(br2)（$(bold)注意：$()饱和度和食物负面效果不会影响）"
```
源码里只取了食物回的饥饿值

---

同意